controller/engine_image_controller_test.go

```diff
@@ -89,7 +89,7 @@ func getEngineImageControllerTestTemplate() *EngineImageControllerTestCase {
 	tc.volume.Status.CurrentNodeID = TestNode1
 	tc.volume.Status.State = longhorn.VolumeStateAttached
 	tc.volume.Status.CurrentImage = TestEngineImage
-	tc.currentEngineImage.Status.RefCount = 1
+	tc.currentEngineImage.Status.RefCount = 2
```

Comment:
nit: OK, because this test only simulates volume and engine w/o replicas, it's 2 here. Probably we should cover all cases here.

Reply:
Yes, we can add test cases containing volumes, engines and replicas in e2e.

Reply:
Let's add test cases for that, but later implementation.

Reply:
NIT: improving the unit test coverage for a change like this, would be nice. We can do it after the release is wrapped up though.